rs/src/management/http_client.rs

```diff
@@ -518,6 +499,33 @@ impl From<TunnelClientBuilder> for TunnelManagementClient {
     }
 }
 
+fn add_query(mut url: Url, tunnel_opts: &TunnelRequestOptions) -> Url
+{
```

Comment:
Please make sure to rust `rustfmt` for formatting. Turning on "format on save" with the rust analyzer extension will do this for you automatically, or you can manually run the "Format Document" command

Reply:
Good call! I ran the formatter and added a note to CONTRIBUTING.md.